src/include/stir/ProjData.h

```diff
@@ -127,6 +128,10 @@ class ProjData : public ExamData
   virtual Succeeded 
     set_sinogram(const Sinogram<float>&) = 0;
 
+  //! construct projection data that stores a subset of the views
+  unique_ptr<ProjData>
```

Comment:
```suggestion
  unique_ptr<ProjDataInMemory>
```
this avoids casting below. This should automatically convert to a `unique_ptr<ProjData>` anyway